sources/packages/api/src-sql/SINValidations/Update-column-student-id.sql

```diff
@@ -0,0 +1,20 @@
+-- Populate the student_id column through the user_id in sims.sin_validations.
+UPDATE
+    sims.sin_validations sv
+SET
+    student_id = (
+        SELECT
+            id
+        FROM
+            sims.students
+        WHERE
+            user_id = sv.user_id
+    );
+
+-- Once the student_id column is populated then make the column not null.
+ALTER TABLE
```

Comment:
I believe that the DEV DB will have users that are associated with more than one student, due to the tests with BCeID upgrade/downgrade, which will make some records on the `sims.sin_validations` not have the student_id associated.

Reply:
As discussed, using the student_users table to populate the student_id column.